addie/addiedriver.py

```diff
@@ -96,7 +96,7 @@ def calculate_gr(self, sq_ws_name, pdf_type, min_r, delta_r, max_r, min_q, max_q
             kwargs['rho0'] = rho0
 
         # get the input unit
-        sq_ws = AnalysisDataService.retrieve(self._currSqWsName)
+        AnalysisDataService.retrieve(self._currSqWsName)
```

Comment:
This line should be removed altogether. It isn't used and isn't needed.

Reply:
Done in 3b5193b

Reply:
3b5193b51d043f674bc98a30891c83e14ee3c870